server/app/controllers/FeatureFlagOverrideController.java

```diff
@@ -0,0 +1,32 @@
+package controllers;
+
+import annotations.FeatureFlagOverrides;
+import auth.Authorizers;
+import javax.inject.Inject;
+import org.pac4j.play.java.Secure;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+
+/**
+ * Allows for overriding of feature flags by the CiviForm Admin via HTTP request.
+ */
+public class FeatureFlagOverrideController extends CiviFormController {
```

Comment:
If possible, can you mark this as `final` to make the decision to allow subclassing explicit rather than implicit?

Reply:
done